src/mlpack/core/optimizers/smorms3/smorms3_update.hpp

```diff
@@ -0,0 +1,111 @@
+/**
+ * @file smorms3_update.hpp
+ * @author Vivek Pal
+ *
+ * SMORMS3 update for Stochastic Gradient Descent.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_SMORMS3_UPDATE_HPP
```

Comment:
Can you use ``#ifndef MLPACK_CORE_OPTIMIZERS_SMORMS3_SMORMS3_UPDATE_HPP`` here, to match the rest of the codebase?